les/clientpool.go

```diff
@@ -301,6 +318,9 @@ func (f *clientPool) connect(peer clientPeer, capacity uint64) bool {
 	totalConnectedGauge.Update(int64(f.connectedCap))
 	clientConnectedMeter.Mark(1)
 	log.Debug("Client accepted", "address", freeID)
+	if f.eventHook != nil {
```

Comment:
It's dangerous to call `eventHook` directly here. Because it involves network IO which can block the entire function.

Let's add a TODO: change the event notification system to non-blocking mode.

Reply:
event subs are removed for now